### PR TITLE
Fix machine export

### PR DESCRIPTION
**Pull request by hatched**
Machine export needs to properly place units based on the bundle v4 spec.

### CODE REVIEW

**Comment by frankban** [comment]
QA ok, thanks a lot!

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2514/
Test FAILed.

File: app/models/models.js [hunk 1]
```diff
@@ -2270,7 +2270,7 @@ YUI.add('juju-models', function(Y) {
           serviceData.num_units = 0;
         } else {
           // Test models or ghosts might not have a units LazyModelList.
-          serviceData.num_units = units && units.size() || 1;
+          serviceData.num_units = units && units.size() || 0;
```
Comment (makyo) [inline app/models/models.js]: Ditto subordinates, unless I'm mistaken.  May need a QA against a bundle with a subordinate service

Comment (hatched) [inline app/models/models.js]: on line 2270 above is where it handles the number of units for subordinates. I believe this was also QA'd to work.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2516/
Test PASSed.

**Comment by makyo** [comment]
:+1:

**Comment by hatched** [comment]
Thanks for the reviews and qa! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**